firmware/development/logic_analyzer.cpp

```diff
@@ -110,14 +110,12 @@ static void initWave(const char *name, int index) {
 	if (brainPin == GPIO_UNASSIGNED)
 		return;
 
-	bool mode = CONFIG(logicAnalyzerMode)[index];
```

Comment:
there wasn't actually any way to set this - should probably remove and save the config space.